gtsfm/utils/metrics.py

```diff
@@ -375,12 +409,16 @@ def compute_ba_pose_metrics(
     metrics = []
     metrics.append(compute_rotation_angle_metric(wRi_aligned_list, gt_wRi_list))
     metrics.append(compute_translation_distance_metric(wti_aligned_list, gt_wti_list))
-    metrics.append(compute_relative_translation_angle_metric(i2Ui1_dict_gt, computed_wTi_list))
+    translation_angular_errors = get_relative_translation_angles(i2Ui1_dict_gt, computed_wTi_list)
+    metrics.append(
+        GtsfmMetric("relative_translation_angle_error_deg", np.array(translation_angular_errors, dtype=np.float32))
+    )
     metrics.append(compute_translation_angle_metric(gt_wTi_list, computed_wTi_list))
-    metrics.append(compute_relative_rotation_angle_metric(i2Ri1_dict_gt, computed_wTi_list))
+    rotation_angular_errors = get_relative_rotation_angles(i2Ri1_dict_gt, computed_wTi_list)
+    metrics.append(
+        GtsfmMetric("relative_rotation_angle_error_deg", np.array(rotation_angular_errors, dtype=np.float32))
+    )
 
-    rotation_angular_errors = metrics[2]._data
```

Comment:
[2] and [4] were swapped here I think, flipping what was passed to AUC metric compute below.